cob_monitoring/src/emergency_stop_monitor.py

```diff
@@ -132,7 +132,7 @@ def jointstate_callback(self, msg):
 	## set light
 	def set_light(self, color, flashing=False):
 		for component in self.light_components:
-			color_rgba = sss.compose_color(component, color)
+			comp, color_rgba = sss.compose_color(component, color)
```

Comment:
If the meaning of the first member of the tuple is "error_code", why not name it `error_code`  instead of `comp`?